mypy.ini

```
@@ -0,0 +1,18 @@
+[mypy]
+files =
+  docs,
+  share,
+  src,
+  tests
```

I just realized that this move dropped the trailing backslashes. It's important to keep them as they were originally used to visually communicate that these are actually folders and not filenames.
```suggestion
  docs/,
  share/,
  src/,
  tests/
```

---

nice.